README.md

```diff
@@ -113,22 +113,25 @@ wrapperUpgrade {
-| `baseBranch`                 | The git branch to checkout and that the pull request will target                                                                                                 |
-| `options.gitCommitExtraArgs` | List of additional git commit arguments                                                                                                                          |
-| `options.allowPreRelease`    | Boolean: true will get the latest Maven/Gradle version even if it's a pre-release. Default is false.                                                             |
-| `options.labels`             | Optional list of label (names) that will be added to the PR.                                                                                                     |
+| Field                                | description                                                                                                                                                               |
+|:-------------------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `name`                               | A name identifying the upgrade, it can be different from the project name, for example when you need to upgrade multiple gradle projects in the same git project          |
+| `repo`                               | The GitHub repository to clone, format 'organization/project`                                                                                                             |
+| `dir`                                | The directory inside the project base directory to run the gradle or maven upgrade in                                                                                     |
+| `baseBranch`                         | The git branch to checkout and that the pull request will target                                                                                                          |
+| `options.gitCommitExtraArgs`         | List of additional git commit arguments                                                                                                                                   |
+| `options.allowPreRelease`            | Boolean: `true` will get the latest Maven/Gradle version even if it's a pre-release. Default is `false`.                                                                  |
+| `options.labels`                     | Optional list of label (names) that will be added to the pull request.                                                                                                    |
+| `options.recreateClosedPullRequests` | Boolean: `true` will recreate the pull request if a closed pull request with the same branch name exists. `false` will not recreate the pull request. Default is `false`. |
```

Comment:
I just stumbled over this while testing it.
Maybe we should remove the last `s`.
Its just a single PR, right?
`options.recreateClosedPullRequest`